mne/report/report.py

```diff
@@ -265,19 +265,22 @@ class _ContentElement:
 
 def _check_tags(tags) -> Tuple[str]:
     # Must be iterable, but not a string
-    if (isinstance(tags, str) or not isinstance(tags, (Sequence, np.ndarray))):
+    if isinstance(tags, str):
+        tags = (tags,)
+    elif isinstance(tags, (Sequence, np.ndarray)):
+        tags = tuple(tags)
+    else:
         raise TypeError(
-            f'tags must be a collection of str, but got {type(tags)} '
-            f'instead: {tags}'
+            f'tags must be a string (without spaces) or an array-like object '
```

Comment:
Whoops, looks like the parenthetical `(without spaces)` should actually go down around line 297. Also I guess it should list the characters `' ', '"', '\n'` since it's not just spaces

Reply:
Should be good now in 2682fd1